pypeit/core/fitting.py

```diff
@@ -924,7 +924,8 @@ def bspline_profile(xdata, ydata, invvar, profile_basis, ingpm=None, upper=5, lo
 
     maskwork = outmask & ingpm & (invvar > 0)
     if not maskwork.any():
-        msgs.error('No valid data points in bspline_profile!.')
+        return [], outmask, yfit, reduced_chi, 1
+        #msgs.error('No valid data points in bspline_profile!.')
```

Comment:
Why was this change made? I am not sure I agree with it. There is no fit if maskwork has no True elements, and so a crash with this error message is probably desired.

Reply:
I had an object for which this line was crashing the pipeline, and I saw other failure modes also simply returned and then went with a different estimation technique. It seemed to allow things to pass OK. Obviously it is easy enough to revert, I had forgotten about it.